samples/texture_3d.html

```diff
@@ -152,16 +186,18 @@
         gl.texImage3D(
             gl.TEXTURE_3D,  // target
             0,              // level
-            gl.R32F,        // internalformat
+            gl.RGB8,        // internalformat
             SIZE,           // width
             SIZE,           // height
             SIZE,           // depth
             0,              // border
-            gl.RED,         // format
-            gl.FLOAT,       // type
+            gl.RGB,         // format
```

Comment:
You could use gl.RED here. That would be closest to the intention of the original sample.